client/state/sites/actions.js

```diff
@@ -14,19 +22,18 @@ import {
 	SITES_REQUEST_FAILURE,
 	SITES_UPDATE
 } from 'state/action-types';
-import { omit } from 'lodash';
 
 /**
  * Returns an action object to be used in signalling that a site has been
  * deleted.
  *
- * @param  {Object} site Site received
+ * @param  {Number} siteId Site id received
```

Comment:
I addressed the description here, but about the alignment issue, looks like you were comparing two mutually exclusive lines. One's out for the other and the return description was already aligned to the parameter.